controllers/grid/admin/languages/AdminLanguageGridHandler.inc.php

```diff
@@ -416,47 +417,33 @@ function setPrimaryLocale($args, $request) {
 	//
 	// Helper methods.
 	//
-	/**
-	 * Update the locale support state (enabled or disabled).
-	 * @param $request Request
-	 * @param $rowId string The locale row id.
-	 * @param $enable boolean Enable locale flag.
-	 */
-	function _updateLocaleSupportState($request, $rowId, $enable) {
```

Comment:
Hmmm... this function is used in enableLocale and disableLocale function...

Reply:
Oops -- adding again, thanks.